velox/functions/prestosql/Slice.cpp

```diff
@@ -200,22 +202,43 @@ class SliceFunction : public exec::VectorFunction {
     return std::min(endIndex - start, length);
   }
 };
+} // namespace
+
+template <TypeKind INDEX_TYPE>
+void registerSliceFunction(const std::string& prefix) {
+  using NativeType = typename TypeTraits<INDEX_TYPE>::NativeType;
+  std::vector<std::shared_ptr<exec::FunctionSignature>> signatures;
+  if constexpr (std::is_same_v<NativeType, int64_t>) {
+    signatures.push_back(exec::FunctionSignatureBuilder()
+                             .typeVariable("T")
+                             .returnType("array(T)")
+                             .argumentType("array(T)")
+                             .argumentType("bigint")
+                             .argumentType("bigint")
```

Comment:
Maybe use TypeTraits<INDEX_TYPE>::name to get the argument type name.

Reply:
done